### DISCUSSION TITLE
What is the correct 'redirect_uri' for oauth when using API Portal

### DISCUSSION THREAD

**Discussion by jdloyd** [mia-platform/community#561] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

1.102.8

### Description

I am trying to enable oAuth authorization in my API portal. I am using the access code flow. I have most everything figured out except what the redirect URL should be in my Okta client app. I've tried a few options with no luck. How do I figure out what redirect URL the API Portal is using when I try to authorize? 

### Actual Outcome

Get a 400 error back from Okta: "Your request resulted in an error. The 'redirect_uri' parameter must be a Login redirect URI in the client app settings"

### Expected Outcome

API portal is able to obtain an JWT to be used with calls to the API

**Comment by LiliumCandidum** [comment]
Hi @jdloyd ,

It seems like you are unable to close the oauth flow, are you using the `login-site` service?

Anyway, can you share more information about your setup and what services are you using for your authentication flow to help you better?

**Comment by jdloyd** [comment, marked as answer]
I did not find an answer to this, but no longer need it to be answered
